code/modules/clothing/masks/muzzle.dm

```diff
@@ -25,3 +25,74 @@
+
+/obj/item/clothing/mask/muzzle/tape
+	name = "tape piece"
+	desc = "A piece of tape that can be put over someone's mouth."
+	icon_state = "tape_piece"
+	worn_icon_state = "tape_piece_worn"
+	inhand_icon_state = null
+	w_class = WEIGHT_CLASS_TINY
+	clothing_flags = INEDIBLE
+	equip_delay_other = 40
+	strip_delay = 40
+	greyscale_config = /datum/greyscale_config/tape_piece
+	greyscale_config_worn = /datum/greyscale_config/tape_piece/worn
+	greyscale_colors = "#B2B2B2"
+	item_flags = NOBLUDGEON
+
+/obj/item/clothing/mask/muzzle/tape/examine(mob/user)
+	. = ..()
+	. += "[span_notice("Target mouth and use it on someone to tape their mouth closed.")]"
+
+/obj/item/clothing/mask/muzzle/tape/attack(mob/living/carbon/victim, mob/living/carbon/attacker, params)
+	if(attacker.zone_selected != BODY_ZONE_PRECISE_MOUTH)
+		to_chat(attacker, span_notice("Target [victim]'s mouth if you want to tape it closed."))
+		return ..()
+	if(victim.is_mouth_covered(head_only = TRUE))
+		to_chat(attacker, span_notice("[victim]'s mouth is covered."))
+		return
+	if(!mob_can_equip(victim, attacker, ITEM_SLOT_MASK))
+		to_chat(attacker, span_notice("[victim] is already wearing somthing on their face."))
+		return
+	balloon_alert(attacker, "taping mouth...")
+	if(!do_after(attacker, equip_delay_other, target = victim))
+		return
+	victim.equip_to_slot_if_possible(src, ITEM_SLOT_MASK)
+	update_appearance()
+
+/obj/item/clothing/mask/muzzle/tape/super
+	name = "super tape piece"
+	desc = "A piece of tape that can be put over someone's mouth. This one has extra strengh."
+	greyscale_colors = "#4D4D4D"
+	strip_delay = 80
+
+/obj/item/clothing/mask/muzzle/tape/surgical
+	name = "surgical tape piece"
+	desc = "A piece of tape that can be put over someone's mouth. As long as you apply this to your patient, you won't hear their screams of pain!"
+	greyscale_colors = "#FFFFFF"
+	equip_delay_other = 30
+	strip_delay = 30
+
+/obj/item/clothing/mask/muzzle/tape/pointy
+	name = "pointy tape piece"
+	desc = "A piece of tape that can be put over someone's mouth. Looks like it will hurt if this is ripped off carelessly."
+	icon_state = "tape_piece_spikes"
+	worn_icon_state = "tape_piece_spikes_worn"
+	greyscale_config = /datum/greyscale_config/tape_piece/spikes
+	greyscale_config_worn = /datum/greyscale_config/tape_piece/worn/spikes
+	greyscale_colors = "#E64539#AD2F45"
+	var/stripping_damage = 10
+
+/obj/item/clothing/mask/muzzle/tape/pointy/doStrip(mob/stripper, mob/living/owner)
+	. = ..()
+	owner.apply_damage(stripping_damage, BRUTE, BODY_ZONE_HEAD)
+	owner.emote("scream")
+	to_chat(owner, span_danger("You feel a massive pain as hundreds of tiny spikes tear free from your face!"))
+
+/obj/item/clothing/mask/muzzle/tape/pointy/super
+	name = "super pointy tape piece"
+	desc = "A piece of tape that can be put over someone's mouth. This thing could rip your face into a thousand pieces if ripped off carelessly."
+	greyscale_colors = "#8C0A00#300008"
+	strip_delay = 60
```

Comment:
intentional that this is faster than regular super tape?

Reply:
Yes, so there's a slight reason to use super sticky tape over super pointy other than it being non-lethal. I also like the idea of using it in a kidnapping situation to make people speak by ripping it off repeatedly.